packages/template-retail-react-app/app/components/product-view/index.jsx

```diff
@@ -125,7 +134,7 @@ const ProductView = forwardRef(
             stockLevel,
             stepQuantity
         } = useDerivedProduct(product, isProductPartOfSet)
-        const {basePrice, discountPrice} = getDisplayPrice(product)
+        const {listPrice, currentPrice} = getDisplayPrice(product)
```

Comment:
This is a good place to implement the `useMemo` hook so we think about performance. I'm not sure how often this component re-renders, hopefully not too much, but ideally we should only run `getDisplayPrice` when the project changes. 

Something like this:

```
        const {listPrice, currentPrice} = useMemo(() => {
            return getDisplayPrice(product)
        }, [product])
```

If there are other uses of this util in components, we'll want to do the same thing there.